_content/booklets/sixty-songs-of-milarepa_milarepa.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Sixty Songs of Milarepa" 
```

Comment:
Sorry, I should have been explicit: if this is poetry then it needs `subcat: poetry` 😊